src/renderer/account/components/TransactionsPanel/TransactionsPanel.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import classNames from 'classnames';
+import { string, objectOf } from 'prop-types';
+
+import Panel from 'shared/components/Panel';
+import Tabs from 'shared/components/Tabs';
+import SendIcon from 'shared/images/account/send.svg';
+import ReceiveIcon from 'shared/images/account/receive.svg';
+
+import Send from './Send';
+import Receive from './Receive';
+import IconTab from './IconTab';
+import balanceShape from '../../shapes/balanceShape';
+import styles from './TransactionsPanel.scss';
+
+const TAB_SEND = 'send';
+const TAB_RECEIVE = 'receive';
+
+const TABS = {
+  [TAB_SEND]: <IconTab renderIcon={SendIcon}>Send</IconTab>,
+  [TAB_RECEIVE]: <IconTab renderIcon={ReceiveIcon}>Receive</IconTab>
+};
+
+export default class TransactionsPanel extends React.PureComponent {
```

Comment:
This name is going to be confusing when you add in the 3rd tab - `Transactions` which would basically also be a `Transactionspanel`

Reply:
Yeah, I'm open to another name for this panel.  Let me know if you have any suggestions!

Reply:
Can't think of something so quickly, maybe `WalletInteractionPanel` or such sounds a little clompy/stupid hahaha